shell/platform/darwin/macos/framework/Source/FlutterEngine.mm

```diff
@@ -264,6 +264,14 @@ - (BOOL)runWithEntrypoint:(NSString*)entrypoint {
   std::transform(switches.begin(), switches.end(), std::back_inserter(argv),
                  [](const std::string& arg) -> const char* { return arg.c_str(); });
 
+  NSArray* entrypointArguments = [_project dartEntrypointArguments];
+  NSEnumerator* enumerator = [entrypointArguments objectEnumerator];
+  NSString* argument;
+  std::vector<const char*> dart_entrypoint_args;
```

Comment:
`dartEntrypointArgs`

Variable name style is decided by the context (here, an ObjC method), not the variable type, per the Google ObjC++ style guidance.